shared/src/iosMain/kotlin/co/nimblehq/ic/kmm/suv/helper/date/DateTimeFormatter.kt

```diff
@@ -0,0 +1,14 @@
+package co.nimblehq.ic.kmm.suv.helper.date
+
+import kotlinx.datetime.LocalDate
+
+actual class DateTimeFormatterImpl: DateTimeFormatter {
+
+    actual override fun getFormattedString(
+        localDate: LocalDate,
+        format: DateFormat
+    ): String {
+        val date = DateFormatter.yearMonthDay.dateFromString(localDate.toString()) ?: return ""
+        return DateFormatter.getBy(format).stringFromDate(date)
+    }
+}
```

Comment:
same for `DateFormatter`, is it for next Integration task?

Reply:
Yes, both platforms need to get `today` information. I believe this can belong to the backend task, so I created the shared code here.